kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_dependency_injection/homework/ShippingCenter.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.spring.basic.spring_dependency_injection.homework;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShippingCenter {
+
+    @Autowired
+    @Qualifier(value = "notificationService")
+    private NotificationService notificationService;
+
+    @Autowired
+    @Qualifier(value = "deliveryService")
```

Comment:
Qualifier jest potrzebny tylko wtedy kiedy jest wiecej niz 1 instancja danej klasy w kontenerze. Tutaj jest zbedny. Z NotificationService to samo